spec/features/social_features_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+feature "Social networking" do
+    given(:jane) { Fabricate(:user) }
+    given(:joe) { Fabricate(:user) }
+    given(:tv_comedies) { Fabricate(:category, name: 'TV Comedies') }
+
+  scenario "following a user" do 
+    futurama = Fabricate(:video, title: 'Futurama', category: tv_comedies)
+    jane_review = Fabricate(:review, video: futurama, user: jane)
+    
+    sign_in(joe)
+    find("a[href='/videos/#{futurama.id}']").click 
+    click_on jane_review.user.full_name
+    click_on 'Follow'
+    expect(current_path).to eq(people_path)
```

Comment:
One more clarification these are your landmarks

```ruby
find("a[href='/videos/#{futurama.id}']").click 
click_on jane_review.user.full_name
click_on 'Follow'
```

You don't need to *assert* on anything further than that for your landmarks.

```ruby
# this is what you have (there is a tractor ride)
expect(current_path).to eq(people_path)
# this is what you need (jane is on the tractor ride)
expect(page).to have_content jane_review.user.full_name
```